File: manager-web/src/main/resources/static/js/service/goodsDescService.js
//服务层
app.service('goodsDescService',function($http){
	    	

	    //查找所有
        this.findAll = function () {
            return $http.get("./goodsDesc");
        }
    
        //分页查找
        this.findPage = function (pageNo, pageSize) {
            return $http.get("./goodsDesc/" + pageNo + "/" + pageSize);
        }
    
        //添加品牌数据
        this.save = function (entity) {
            return $http.post("./goodsDesc", entity);
        }
    
        //修改品牌数据
        this.update = function (entity) {
            return $http.put("./goodsDesc", entity);
        }
    
        //删除品牌数据
        this.delete = function (selectIds) {
            return $http.delete("./goodsDesc?ids=" + selectIds);
        }
    
        //模糊查询品牌数据
        this.search=function (pageNo, pageSize,entity) {
            return $http.post("./goodsDesc/search/" + pageNo + "/" + pageSize, entity);
        }
  	
});

File: sellergoods-interface/src/main/java/com/gm/service/GoodsDescService.java
package com.gm.service;

import com.gm.common.PageResult;
import com.gm.common.Result;
import com.gm.pojo.TbGoodsDesc;

import java.util.List;

public interface GoodsDescService {
    Result findAll();

    PageResult findPage(Integer pageNo, Integer pageSize);

    Result findPage(TbGoodsDesc goodsDesc, Integer pageNo, Integer pageSize);

    Result saveGoodsDesc(TbGoodsDesc goodsDesc);

    Result updateGoodsDesc(TbGoodsDesc goodsDesc);

    Result deleteGoodsDesc(Long[] ids);

    Result deleteOneGoodsDesc(Long id);
}
